explainer.md

```diff
@@ -17,40 +17,33 @@ The two drafts apply to both TLS server certificates, where the server authentic
 
-Today, subscribers typically provision a single certificate for all supported relying parties, because relying parties do not communicate which CAs are trusted. We call this a *single-certificate deployment model*. In this model, the single certificate must simultaneously satisfy all relying parties.
+A single subscriber may need to interoperate with relying parties that trust different sets of CAs. To accommodate this, TLS 1.3 defines the [`certificate_authorities` extension](https://www.rfc-editor.org/rfc/rfc8446#section-4.2.4).This allows the subscriber to provision multiple certificates and select the one that will allow the relying party to accept its TLS key. This is analogous to parameter negotiation elsewhere in TLS.
 
-This constraint imposes costs on the ecosystem as PKIs evolve over time. The older the relying party, the more its requirements may have diverged from newer ones, making it increasingly difficult for subscribers to support both. This translates to analogous costs for CAs and relying parties:
+However, this extension's size is impractical for some applications. Existing PKIs may have many CAs, and existing CAs may have long X.509 names. As of August 2023, the Mozilla CA Certificate Program {{MOZILLA-ROOTS}} contained 144 CAs, with an average name length of around 100 bytes. While applications could mitigate this with smaller PKIs and transitioning to newer CAs with shorter names, this is incompatible with existing PKIs.
 
-* For a new CA to be usable by subscribers, it must be trusted by all relying parties. This is particularly challenging for older, unupdatable relying parties. Existing CAs face similar challenges when rotating or deploying new keys.
+Without a negotiation mechanism, the subscriber must instead obtain a single certificate which simultaneously satisfies all relying parties. This is a challenge for subscribers when relying parties are diverse. PKI evolution naturally leads to relying party diversity, so this translates to analogous challenges for CAs and relying parties:
 
-* When a relying party must update its policies to meet new security requirements, it must choose between compromising on user security or imposing a significant burden on subscribers that still support older relying parties.
-
-The two drafts aim to remove this constraint, by enabling a *multi-certificate deployment model*. Subscribers are instead provisioned with multiple certificates and automatically select the correct one to use with each relying party. This allows a single subscriber to use different certificates for different relying parties, including older and newer ones.
-
-There are three parts to understanding this proposal:
+* For a new CA to be usable by subscribers, all relying parties must trust it. This is particularly challenging for older, un-updatable relying parties. Existing CAs face similar challenges when rotating or deploying new keys.
```

Comment:
Before a new key can be used by a subscriber, all relying parties must trust it. This prevents new keys from being used if a subscriber must support older un-updatable relying parties, or before all relying parties agree to trust the key.  This obstacle to using new keys prevents new CA's from entering the ecosystem, or existing CA's from rotating or deploying new keys.

Reply:
I think "new CA" is correct, not "new key". "New key" reads like it's specifically talking about key rotation, but this is a summary of the general "new CA" problem. (CA != CA operator.) But yeah this (existing) line is a little bit hard to read. I'll take a pass and incorporate bits of your text.